csharpcore/GildedRoseTests/GildedRoseTest.cs

```diff
@@ -1,18 +1,65 @@
-﻿using Xunit;
-using System.Collections.Generic;
+﻿using System.Collections.Generic;
 using GildedRoseKata;
+using NUnit.Framework;
 
 namespace GildedRoseTests
 {
+    [TestFixture]
     public class GildedRoseTest
     {
-        [Fact]
-        public void foo()
+        [TestFixture]
+        public class UpdateQuality
         {
-            IList<Item> Items = new List<Item> { new Item { Name = "foo", SellIn = 0, Quality = 0 } };
-            GildedRose app = new GildedRose(Items);
-            app.UpdateQuality();
-            Assert.Equal("fixme", Items[0].Name);
+            [TestFixture]
+            public class ForEachStandardItem
+            {
+                [Test]

```

Comment:
With NUnit, you need the attribute [Test] to declare a function as test.
You can also use [TestCase(samples)] to run test cases